lms/djangoapps/instructor/views/api.py

```diff
@@ -3380,3 +3428,25 @@ def _get_boolean_param(request, param_name):
     values to boolean values.
     """
     return request.POST.get(param_name, False) in ['true', 'True', True]
+
+
+def _use_json(request):
+    """
+    Returns whether the current request expects a response in JSON form.
+    """
```

Comment:
I'm going to get rid of it. I don't think there are actually any cases where we don't want to return JSON, you're right.